src/main/java/com/superdevs/HealthOMeter/entity/Contact.java

```diff
@@ -0,0 +1,232 @@
+package com.superdevs.HealthOMeter.entity;
+
+import javax.validation.constraints.NotBlank;
+
+import javax.persistence.*;
+import java.time.LocalDateTime;
+import java.util.Set;
+
+@Entity(name = "contacts")
+public class Contact {
+
+    @Id
+    @GeneratedValue(strategy = GenerationType.IDENTITY)
+    @Column(name = "id")
+    private long id;
```

Comment:
I am really not fond of this type of column name. I believe that it will cause an error (not sure tho).
@Column(name="contact_id")

This is what I do recommend. It applies to all sections. |

Przemek, what are your thoughts?

Reply:
when You mapped You are use contact_id. If this is PK, it is just "'id" - when You look at  Table You already know that is Contact ID